covid19-etl/etl/idph_facility.py

```diff
@@ -159,7 +132,7 @@ def parse_facility(self, date, facility):
             "deaths": deaths,
             "submitter_id": summary_clinical_submitter_id,
             "lastUpdateEt": date,
-            "date": date,
+            "date": etlJobDate,
```

Comment:
i might have misunderstood the conversation today but i would do the opposite;
`lastUpdateEt = today (the last date this data was updated)`
and `date = facility["ReportDate"] (the date the data is for)`

Reply:
so, the `lastUpdateEt` must be the `etlJobDate`?

Reply:
see Sean's message on slack:
"If you have an entry that is on 2020-11-13, then `date` is `2020-11-13`, but the `lastUpdatedEt` is `2021-08-23`."